app/models/deed_type.rb

```diff
@@ -0,0 +1,63 @@
+# A DeedType describes the kind of Deed a user has contributed to a page. This may
+# be a transcription, translation, page edit, etc.
+
+class DeedType
+  # These constants are called individually in several parts of the app for specific
+  # deed types. Their values are stored as identifiers in the deeds.deed_type table field.
+  PAGE_TRANSCRIPTION = 'page_trans'
+  PAGE_EDIT = 'page_edit'
+  PAGE_INDEXED = 'page_index'
+  PAGE_MARKED_BLANK = 'markd_blnk'
+  ARTICLE_EDIT = 'art_edit'
+  NOTE_ADDED = 'note_add'
+  PAGE_TRANSLATED = 'pg_xlat'
+  PAGE_TRANSLATION_EDIT = 'pg_xlat_ed'
+  OCR_CORRECTED = 'ocr_corr'
+  NEEDS_REVIEW = 'review'
+  TRANSLATION_REVIEW = 'xlat_rev'
+  TRANSLATION_INDEXED = 'xlat_index'
+  WORK_ADDED = 'work_add'
+
+  # The TYPES hash houses all of the deed types and makes is easier to access
+  # groups of deed types and also their human-readable names. Any new deed type
+  # constant should also be added here.
+  TYPES = {
+    "#{PAGE_TRANSCRIPTION}" => 'Page Transcribed',
+    "#{PAGE_EDIT}" => 'Page Edited',
+    "#{PAGE_INDEXED}" => 'Page Indexed',
+    "#{PAGE_MARKED_BLANK}" => 'Page Marked Blank',
+    "#{ARTICLE_EDIT}" => 'Article Edited',
+    "#{NOTE_ADDED}" => 'Note Added',
+    "#{PAGE_TRANSLATED}" => 'Page Translated',
+    "#{PAGE_TRANSLATION_EDIT}" => 'Translation Page Edited',
+    "#{OCR_CORRECTED}" => 'Page OCR Corrected',
+    "#{NEEDS_REVIEW}" => 'Page Needs Review',
+    "#{TRANSLATION_REVIEW}" => 'Translation Page Needs Review',
+    "#{TRANSLATION_INDEXED}" => 'Translation Page Indexed',
+    "#{WORK_ADDED}" => 'Work Added'
+  }
+
```

Comment:
Can we get rid of the variable interpolation for the hash keys here?  
Both `"#{WORK_ADDED}"` and `WORK_ADDED` will evaluate to `'work_add'` and the latter is less complex.

Reply:
What?! 😮That's awesome! I'm on it.